### PR TITLE
Add PHPCS PSR-12 support to docker flex

### PR DESCRIPTION
**Pull request by prondubuisi**
supports flex 3.10,3.11,edge
fixes issue #247

### CODE REVIEW

File: docker/openemr/flex-3.10/utilities/devtools [hunk 1]
```diff
@@ -12,18 +12,24 @@ if [ "$1" == "build-themes" ]; then
...
-    echo "Generating PSR2 code styling error report "
+if [ "$1" == "psr2-report"  ] || [ "$1" == "psr12-report"  ]; then
+    echo "Generating PSR12 code styling error report "
     cd /var/www/localhost/htdocs/openemr
     php -d memory_limit=640M /root/.composer/vendor/squizlabs/php_codesniffer/bin/phpcs -n --extensions=php,inc --standard=ci/phpcs.xml --report=full .
 fi
 
-if [ "$1" == "psr2-fix"  ] || [ "$1" == "clean-sweep" ]; then
-    echo "Fixing PSR2 code styling errors "
+if [ "$1" == "psr2-fix"  ] ||  [ "$1" == "psr12-fix"  ] ||[ "$1" == "clean-sweep" ]; then
```
Comment (bradymiller) [inline docker/openemr/flex-3.10/utilities/devtools:21]: hi @prondubuisi 
bash can be temperamental with spacing on conditionals, so rec having a space above between the `||` and `[`

**Comment by bradymiller** [comment]
just noted a minor issue. otherwise looks good.

File: docker/openemr/flex-3.10/utilities/devtools (hunk 1 above)
Comment (prondubuisi) [reply docker/openemr/flex-3.10/utilities/devtools]: Fixed @bradymiller thanks. I need those Eagle eyes of yours!

**Comment by prondubuisi** [comment]
> just noted a minor issue. otherwise looks good.

Fixed!

**Comment by bradymiller** [comment]
hi @prondubuisi , Looks good and bringing this in. Plan to rebuild dockers soon. thanks! -brady

**Pull request merged**